src/API/firstAPI.html

```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
+<html>
+<head>
+<meta charset="UTF-8">
+<title>API</title>
+</head>
+
+<body>
+<div id="imagesContainer">Here is the div!
+<p>Take a look at <a href="https://jsonplaceholder.typicode.com/photos" target="_blank">https://jsonplaceholder.typicode.com/photos</a></p>
+</div>
+
+<script src = "firstAPI.js"></script>
+</body>
+
```

Comment:
The tab indentation of the firstApi.html and index.html files is broken, can you fix it?